plugins/modules/idrac_network_attributes.py

```diff
@@ -0,0 +1,743 @@
+      - Provides the option to wait for job completion.
+      - This is applicable when I(apply_time) is C(Immediate) for I(oem_network_attributes).
+  job_wait_timeout:
+    type: int
+    default: 1200
+    description:
+      - The maximum wait time of I(job_wait) in seconds. The job is tracked only for this duration.
+      - This option is applicable when I(job_wait) is C(true).
+requirements:
+    - "python >= 3.9.6"
+author:
+    - "Abhishek Sinha(@ABHISHEK-SINHA10)"
+notes:
+    - Run this module from a system that has direct access to Dell iDRAC.
+    - This module supports both IPv4 and IPv6 address.
+    - This module supports C(check_mode).
+"""
+
+EXAMPLES = """
+---
+- name: Configure OEM network attributes
+  dellemc.openmanage.idrac_network_attributes:
+    idrac_ip: "192.168.0.1"
+    idrac_user: "user_name"
+    idrac_password: "user_password"
+    network_id: "NIC.Integrated.1"
+    network_port_id: "NIC.Integrated.1-1-1"
+    oem_network_attributes:
+      IscsiInitiatorIpAddr: "192.168.1.0"
+
+- name: Configure OEM network attributes to apply on reset
+  dellemc.openmanage.idrac_network_attributes:
+    idrac_ip: "192.168.0.1"
+    idrac_user: "user_name"
+    idrac_password: "user_password"
+    network_id: NIC.Integrated.1
+    network_port_id: "NIC.Integrated.1-1-1"
+    oem_network_attributes:
+      SNMP.1.AgentCommunity: public
+    apply_time: OnReset
+    maintenance_window:
+      start_time: "2022-09-30T05:15:40-05:00"
+      duration: 600
+
+- name: Configure OEM network attributes to apply at maintainance window
+  dellemc.openmanage.idrac_network_attributes:
+    idrac_ip: "192.168.0.1"
+    idrac_user: "user_name"
+    idrac_password: "user_password"
+    network_id: NIC.Integrated.1
+    network_port_id: "NIC.Integrated.1-1-1"
+    oem_network_attributes:
+      SNMP.1.AgentCommunity: public
+    apply_time: AtMaintenanceWindowStart
+    maintenance_window:
+      start_time: "2022-09-30T05:15:40-05:00"
+      duration: 600
+
+- name: Clearing the pending attributes
+  dellemc.openmanage.idrac_network_attributes:
+    idrac_ip: "192.168.0.1"
+    idrac_user: "user_name"
+    idrac_password: "user_password"
+    network_id: NIC.Integrated.1
+    network_port_id: "NIC.Integrated.1-1-1"
+    clear_pending: true
+
+- name: Configure OEM network attributes and wait for the job
+  dellemc.openmanage.idrac_network_attributes:
+    idrac_ip: "192.168.0.1"
+    idrac_user: "user_name"
+    idrac_password: "user_password"
+    network_id: NIC.Integrated.1
+    network_port_id: "NIC.Integrated.1-1-1"
+    oem_network_attributes:
+      IscsiInitiatorIpAddr: "192.168.1.0"
+    job_wait: true
+    job_wait_timeout: 2000
+
+- name: Configure redfish network attributes to update fiber channel on reset
+  dellemc.openmanage.idrac_network_attributes:
+    idrac_ip: "192.168.0.1"
+    idrac_user: "user_name"
+    idrac_password: "user_password"
+    network_id: NIC.Integrated.1
+    network_port_id: "NIC.Integrated.1-1-1"
```

Comment:
can we update these examples to include apply_time.
since its a reqd field.

Reply:
Done. Thanks for pointing out.